maint/Makefile_header.PL

```diff
@@ -0,0 +1,43 @@
+use Config;
```

Comment:
Interesting.  I may steal this for one of my modules that I was having issues moving to Dist::Zilla